inference-engine/tests/functional/plugin/gna/pass_tests/decompose_mvn.cpp

```diff
@@ -0,0 +1,157 @@
+        mvnSpecificParams mvnParams;
+        InferenceEngine::Precision netPrecision;
+        std::string targetDevice;
+        std::map<std::string, std::string> configuration;
+        InferenceEngine::SizeVector inputShape;
+        std::tie(mvnParams, netPrecision, targetDevice, configuration, inputShape) = obj.param;
+        float eps;
+        op::MVNEpsMode epsMode;
+        bool normalizeVariance, acrossChannels, mvnVersion6;
+        std::tie(normalizeVariance, eps, epsMode, acrossChannels, mvnVersion6) = mvnParams;
+
+        std::ostringstream result;
+        result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+        result << "NV=" << normalizeVariance << "_";
+        result << "eps=" << eps << "_";
+        result << "mode=" << static_cast<uint32_t>(epsMode) << "_";
+        result << "AC=" << acrossChannels << "_";
+        result << "version=" << mvnVersion6 << "_";
+        result << "netPRC=" << netPrecision.name() << "_";
+        result << "targetDevice=" << targetDevice << "_";
+        for (auto const& configItem : configuration) {
+            result << "_configItem=" << configItem.first << "_" << configItem.second;
+        }
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        threshold = 0.2f;
+        mvnSpecificParams mvnParams;
+        InferenceEngine::Precision netPrecision;
+        InferenceEngine::SizeVector inputShape;
+        std::tie(mvnParams, netPrecision, targetDevice, configuration, inputShape) = this->GetParam();
+        float eps;
+        op::MVNEpsMode epsMode;
+        bool normalizeVariance, acrossChannels, mvnVersion6;
+        std::tie(normalizeVariance, eps, epsMode, acrossChannels, mvnVersion6) = mvnParams;
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto input = builder::makeParams(ngPrc, {inputShape});
+        InferenceEngine::SizeVector axes(inputShape.size() - 2);
+        std::iota(axes.begin(), axes.end(), 2);
+        std::shared_ptr<ngraph::Node> mvn;
+
+        if (mvnVersion6) {
+            const auto axesConst = std::make_shared<op::v0::Constant>(element::i64, Shape{axes.size()}, axes);
+            mvn = std::make_shared<opset7::MVN>(input[0], axesConst, normalizeVariance, eps, epsMode);
+        } else {
+            mvn = std::make_shared<opset2::MVN>(input[0], acrossChannels, normalizeVariance);
+        }
+
+        auto result = std::make_shared<Result>(mvn);
+        function = std::make_shared<Function>(ResultVector{result}, ParameterVector{input});
+    }
+};
+
+TEST_P(DecomposeMVNTest, CompareWithRefs) {
+    Run();
+}
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16
+};
+
+const std::vector<std::map<std::string, std::string>> configs = {
+    {
+        {"GNA_DEVICE_MODE", "GNA_SW_FP32"},
```

Comment:
What about SW exact mode?

Reply:
It was decided that SW_FP32 alone should be acceptable for now due to accuracy of results with SW_EXACT.